### DISCUSSION TITLE
Can't for the life of me figure out how to get slot arrows

### DISCUSSION THREAD

**Discussion by raisedeyebrowemoji** [TodePond/Arroost#303] [Q&A]
Tried connecting arrow of creation to literally anything, it either duplicates that arrow, creates arrow of recording or softlocks the game. Same thing goes for arrow of time. How do you get them?

**Comment by TodePond** [comment]
Slot is an internal thing only. Apart from softlocking, then arrow of creation is working as intended.

Can you tell me exact steps to cause it to softlock?

Arrow of time is also an internal thing, used in wires.
